index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <!-- navgator -->
+
+      <nav>
+        <div class="logo">
+          <a href="#">Welcome</a>
+        </div>
+
+        <!-- menu hamburger -->
+        <a href="#"><img class="menu" src="imege/Enabled.png" alt="Menu" /></a>
+      </nav>
+
+      <!-- middle section -->
+
+      <div class="middle">
+        <!-- Header -->
+
+        <header id="title">
+          <div class="title-content">
+            <h1 id="head-1">Hey there. I’m Muhmmd.</h1>
+            <h1 id="head-2">I’m a software developer</h1>
+          </div>
+        </header>
+
+        <!-- info section -->
+
+        <section class="info">
+          <h4>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </h4>
+          >
+        </section>
+
+        <!-- footer -->
+
+        <footer>
+          <ul>
+            <li>
+              <a href="https://twitter.com/MuhmmdUsama" target="blank"
+                ><img src="imege/twitter.png" alt="social logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://github.com/MuhmmdUsama" target="blank"
+                ><img src="imege/github.png" alt="social logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/muhmmdusama/" target="blank"
+                ><img src="imege/Linkedin-icon.png" alt="social logo"
+              /></a>
+            </li>
+          </ul>
+        </footer>
+      </div>
```

Comment:
Please make sure that this section is arranged that way:

- [ ] Kindly add all content from `"Hey there. I’m Muhmmd."` to social media icons inside `<section>` tag.
- [ ] please add `I’m a software developer` inside `<h2>` tag instead of `<h1>` because you only allowed to use `<h1>` tag one time per page.
- [ ] the paragraph between 39-43 lines should be inside a `<p>` tag so please fix that.
- [ ] Nice implementation of social icons you doing it the right way inside `ul` but please remove it from the `<footer>` tag and just add it inside the `<section>` as I mentioned before.